mlrun/api/api/endpoints/model_endpoints.py

```diff
@@ -228,7 +226,7 @@ def list_model_endpoints(
     project: str,
     model: Optional[str] = Query(None),
     function: Optional[str] = Query(None),
-    labels: List[str] = Query([], alias="label"),
+    labels=Query([], alias="label"),
```

Comment:
In this PR, the labels can be either a list a string (that will be converted later into a dict). However, updating the type hint accordingly
`labels: Union[List[str], str] = Query([], alias="label")`
raise an `AssertionError` when redeploying the mlrun api service:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/mlrun/mlrun/api/main.py", line 32, in <module>
    from mlrun.api.api.api import api_router
  File "/mlrun/mlrun/api/api/api.py", line 19, in <module>
    from mlrun.api.api.endpoints import (
  File "/mlrun/mlrun/api/api/endpoints/model_endpoints.py", line 225, in <module>
    def list_model_endpoints(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 630, in decorator
    self.add_api_route(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 569, in add_api_route
    route = route_class(
  File "/usr/local/lib/python3.9/site-packages/fastapi/routing.py", line 438, in __init__
    self.dependant = get_dependant(path=self.path_format, call=self.endpoint)
  File "/usr/local/lib/python3.9/site-packages/fastapi/dependencies/utils.py", line 323, in get_dependant
    assert isinstance(
AssertionError: Param: labels can only be a request body, using Body()
```
I don't think it's so urgent to keep the type hint in this case since anyway we are planning to support only str type from `1.4.0` and then it should work with the type hint.